themes/Gruntfile.js

```diff
@@ -0,0 +1,266 @@
+'use strict';
+
+const path = require('path');
+const sass = require('sass');
+const autoprefixer = require('autoprefixer');
+const pkg = require('./package.json');
+
+const SWATCHES = [
+  'cerulean',
+  'cosmo',
+  'cyborg',
+  'darkly',
+  'flatly',
+  'journal',
+  'litera',
+  'lumen',
+  'lux',
+  'materia',
+  'minty',
+  'morph',
+  'pulse',
+  'quartz',
+  'sandstone',
+  'simplex',
+  'sketchy',
+  'slate',
+  'solar',
+  'spacelab',
+  'superhero',
+  'united',
+  'vapor',
+  'yeti',
+  'zephyr'
+];
+
+const BUILD_DIR = 'build/';
+const DIST_DIR = 'themes/';
+const DOCS_DEST = 'docs/5/';
```

Comment:
I think that we can delete all things related to docs as we don't aim at redistributing a full modified bootstrap copy.